framework/base/variables/_spacing.scss

```diff
@@ -2,6 +2,7 @@
 
 $ecl-spacing: (
   none: 0,
+  xxxs: 0.33rem,
```

Comment:
More or less 6px then :) After shipping the first version, we should open the discussion with the design team about normalizing those values